src/openfl/_internal/renderer/opengl/batcher/BlendMode.hx

```diff
@@ -0,0 +1,62 @@
+package openfl._internal.renderer.opengl.batcher;
+
+import lime.graphics.GLRenderContext;
+import lime.graphics.opengl.GL;
+import openfl.display.BlendMode as OpenFLBlendMode;
+
+class BlendMode {
+	public static var NONE(default,never) = new BlendMode(GL.FUNC_ADD, GL.ONE, GL.ZERO);
+	public static var NORMAL(default,never) = new BlendMode(GL.FUNC_ADD, GL.ONE, GL.ONE_MINUS_SRC_ALPHA);
+	public static var ADD(default,never) = new BlendMode(GL.FUNC_ADD, GL.ONE, GL.ONE);
+	public static var MULTIPLY(default,never) = new BlendMode(GL.FUNC_ADD, GL.DST_COLOR, GL.ONE_MINUS_SRC_ALPHA);
+	public static var SCREEN(default,never) = new BlendMode(GL.FUNC_ADD, GL.ONE, GL.ONE_MINUS_SRC_COLOR);
+	public static var SUBTRACT(default,never) = new BlendMode(GL.FUNC_REVERSE_SUBTRACT, GL.ONE, GL.ONE);
+	public static var ERASE(default,never) = new BlendMode(GL.FUNC_ADD, GL.ZERO, GL.ONE_MINUS_SRC_ALPHA);
+	public static var MASK(default,never) = new BlendMode(GL.FUNC_ADD, GL.ZERO, GL.SRC_ALPHA);
+	public static var BELOW(default,never) = new BlendMode(GL.FUNC_ADD, GL.ONE_MINUS_DST_ALPHA, GL.DST_ALPHA);
+
+	// public static var NOPREMULT_NONE(default,never) = new BlendMode(GL.FUNC_ADD, GL.ONE, GL.ZERO);
```

Comment:
I guess this is mine, but I don't remember why it's here, probably better to remove these commented lines :)

Reply:
Done!